src/components/single-product/SingleProduct.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+import { useData } from "../../store/data/DataContext";
+import { MdLocalOffer } from "react-icons/md";
+import { useAuth } from "../../store/data/AuthContext";
+import { useCart } from "../../store/data/CartContext";
+import { useWishlist } from "../../store/data/WishlistContext";
```

Comment:
try using the index file in the store folder. you can import all in one line

Reply:
sure